01.general/17.dns-providers/docs.en.md

```diff
@@ -70,6 +98,21 @@ These servers provide the Default features + Blocking adult websites + Safe sear
 | DNSCrypt, IPv6 | Provider: `2.dnscrypt.family.ns2.adguard.com` IP: `[2a00:5a60::bad2:0ff]:5443`| <a href="sdns://AQIAAAAAAAAAGlsyYTAwOjVhNjA6OmJhZDI6MGZmXTo1NDQzIIwhF6nrwVfW-2QFbwrbwRxdg2c0c8RuJY2bL1fU7jUfITIuZG5zY3J5cHQuZmFtaWx5Lm5zMi5hZGd1YXJkLmNvbQ">Add to AdGuard</a> |
 | DNS-over-HTTPS | `https://dns-family.adguard.com/dns-query` | <a href="sdns://AgcAAAAAAAAAAAAWZG5zLWZhbWlseS5hZGd1YXJkLmNvbQovZG5zLXF1ZXJ5">Add to AdGuard</a> |
 | DNS-over-TLS | `tls://dns-family.adguard.com` | <a href="sdns://AwAAAAAAAAAAAAAcdGxzOi8vZG5zLWZhbWlseS5hZGd1YXJkLmNvbQ">Add to AdGuard</a> |
+| DNS-over-QUIC | `quic://dns-family.adguard.com` | <a href="sdns://AwAAAAAAAAAAAAAdcXVpYzovL2Rucy1mYW1pbHkuYWRndWFyZC5jb20">Add to AdGuard</a> |
```

Comment:
See https://github.com/AdguardTeam/AdGuardHome/issues/2049
However i removed DoQ rows and not intend for move to separate PR because that's simple thing, someone in adguard may able to do that. (only adguard dns offer that currently)